crates/bench/src/sqlite.rs

```diff
@@ -1,13 +1,36 @@
-use crate::prelude::*;
-use rusqlite::{Connection, Transaction};
+use crate::{
+    database::BenchDatabase,
+    schemas::{table_name, BenchTable, TableStyle},
+    ResultBench,
+};
+use rusqlite::Connection;
+use spacetimedb_lib::{
+    sats::{self},
+    AlgebraicType, AlgebraicValue, ProductType,
+};
+use std::{fmt::Write, hint::black_box};
+use tempdir::TempDir;
 
-impl BuildDb for Connection {
-    fn build(prefill: bool, fsync: bool) -> ResultBench<Self>
+pub struct SQLite {
+    db: Connection,
+    _temp_dir: TempDir,
```

Comment:
The finickiness of the TempDir drop stuff makes me want a comment here, we've had enough bugs in tests and CI about it that I don't want this refactored away to a local or whatever.

Reply:
In fact I think refactoring it to a local could cause the tempdir to be dropped, which might break things... then again, the earlier benchmarks version did that and the sqlite benches still worked. So idk.